go.sum

```diff
@@ -713,8 +720,8 @@ kubevirt.io/containerized-data-importer v1.10.9 h1:wYCPyMbCLdn5tlZWW/lPYTMPtIOSf
 kubevirt.io/containerized-data-importer v1.10.9/go.mod h1:qF594BtRRkruyrqLwt3zbLCWdPIQNs1qWh4LR1cOzy0=
 kubevirt.io/qe-tools v0.1.6 h1:S6z9CATmgV2/z9CWetij++Rhu7l/Z4ObZqerLdNMo0Y=
 kubevirt.io/qe-tools v0.1.6/go.mod h1:PJyH/YXC4W0AmxfheDmXWMbLNsMSboVGXKpMAwfKzVE=
-libvirt.org/libvirt-go v5.0.0+incompatible h1:0kjba9uMMwHm036eU20ys53WFFn9KxcHC/ps9gEjlWA=
-libvirt.org/libvirt-go v5.0.0+incompatible/go.mod h1:CPoljLoiC2aEw+62g1rZXl2oXAJaNsrq4YCSmJOELek=
+libvirt.org/libvirt-go v6.5.0+incompatible h1:7hrVIm1onDKjIYpKyWqAtF6+ugYOVSnyaWnNpn+8tLI=
+libvirt.org/libvirt-go v6.5.0+incompatible/go.mod h1:CPoljLoiC2aEw+62g1rZXl2oXAJaNsrq4YCSmJOELek=
```

Comment:
Oops wait a moment. This is a big change! is it needed by this PR?

Reply:
@stu-gott  Yes afaik, I need a libvirt-go which includes my [patch.](https://gitlab.com/libvirt/libvirt-go/-/commit/a0815d68d2c703ca2e64d61ba2079db03fa4f4fd). It's tagged on 6.5.0.